scapy/arch/pcapdnet.py

```diff
@@ -347,12 +366,14 @@ def send(self, x):
 
                 def __del__(self):
                     pcap.close(self.pcap)
-            open_pcap = lambda *args, **kargs: _PcapWrapper_libpcap(*args, **kargs)
+            open_pcap = lambda *args, **kargs: _PcapWrapper_libpcap(
+                *args, **kargs)
```

Comment:
I don’t really like the way it get fixed. For instance, I find it way more understandable the way it was previously on this line...

Reply:
In such cases, I'd rather have the closing parenthesis after another line break:
```Python
        open_pcap = lambda *args, **kargs: _PcapWrapper_libpcap(
            *args, **kargs
        )
```

Reply:
`autopep8` does not do it this way. I could try to find another tool, but it might be a pain.